app/components/press-release/form.js

```diff
@@ -52,8 +53,29 @@ export default class PressReleaseFormComponent extends Component {
     this.args.pressRelease.publicationChannels = selectedChannels;
   }
 
+  @action
+  setSources(sources) {
+    this.args.pressRelease.sources = sources;
+    this.showSourceModal = false;
+  }
+
+  @action
+  removeSource(sources) {
+    this.args.pressRelease.sources = sources;
+  }
```

Comment:
Een meer gebruikte interface voor verwijderen is dat het object dat verwijderd moet worden meegegeven wordt als argument en hier verwijderd wordt uit de lijst van `this.args.pressRelease.sources` (dus de logica die je nu eigenlijk uitvoert in `press-release/sources-table.js`)